pkg/lightning/backend/local/local.go

```diff
@@ -509,12 +509,50 @@ func (c *BackendConfig) adjust() {
 	c.MaxOpenFiles = max(c.MaxOpenFiles, openFilesLowerThreshold)
 }
 
-// Backend is a local backend.
-type Backend struct {
+// Clients is the set of clients for creating a local backend
+type Clients struct {
```

Comment:
`Clients` is so common term, we don't know what's inside from the name, and besides, what's the purpose to have such struct

Reply:
Rename to `BackendClients` [`b059c04` (#63467)](https://github.com/pingcap/tidb/pull/63467/commits/b059c04015db953a3e9908e54aad4cb95d4c645d)